cmake/winml_unittests.cmake

```diff
@@ -59,10 +59,11 @@ add_dependencies(winml_test_common
   winml_api
   winml_dll
 )
-target_compile_definitions(winml_test_common PRIVATE BUILD_GOOGLE_TEST)
-set_winml_target_properties(winml_test_common)
 
+set_winml_target_properties(winml_test_common)
 file(GLOB winml_test_api_src CONFIGURE_DEPENDS "${WINML_TEST_SRC_DIR}/api/*.cpp")
+# set global variable for the api test source list for other test cmake files to use
+set(winml_test_api_src_list ${winml_test_api_src} PARENT_SCOPE)
```

Comment:
>PARENT_SCOPE [](start = 50, length = 12)

worried that this is a little brittle as it is dependent on the include structure of cmakelist.txt.

I believe functions are global scoped - can we try getting the list of files with an out param. 

the cppwinrt sdk helpers have examples of out params in cmake.

Reply:
sounds good!